docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -128,14 +130,14 @@ The Academic Twitter API allows you to search all tweets, not just the last 7 da
 using `client.search_all()` function from `twarc` package. To get all tweets (without comments) for our query,
  run the following:
 
-```python
+```bash linenums="1" hl_lines="4-4" title="python"
 search_results = client.search_all(query=query, start_time=start_time, end_time=end_time, max_results=100)
 ```
 
 Once we examine the tweets, we can decide what fields we want to keep and then construct a handy dataframe to store
 the tweets:
 
-```python
+```bash linenums="1" hl_lines="1-1" title="python"
```

Comment:
the language is python and we don't need to highlight

Reply:
done